tracer/src/Datadog.Trace/Ci/Agent/CIWriterHttpSender.cs

```diff
@@ -120,7 +120,7 @@ private async Task SendPayloadAsync<T>(Uri url, Func<IApiRequest, T, Task<IApiRe
                 {
                     exception = ex;
 
-                    if (_globalSettings.DebugEnabled)
+                    if (_isDebugEnabled)
```

Comment:
Nit: Do we really need this field, now? Can't we just do 
```suggestion
                    if (GlobalSettings.Instance.DebugEnabled)
```

Reply:
Total nit though, take it or leave it :)